report/renderer/report-ui-features.js

```diff
@@ -293,7 +282,7 @@ export class ReportUIFeatures {
       const thirdPartyRows = this._getThirdPartyRows(rowEls, this.json.finalUrl);
 
       // create input box
-      const filterTemplate = this._dom.cloneTemplate('#tmpl-lh-3p-filter', this._templateContext);
+      const filterTemplate = this._dom.createComponent('3pFilter');
```

Comment:
TODO: i started renaming things to `component` everywhere (here would be `filterComponent`) but I did not follow through with every file yet

Reply:
> TODO: i started renaming things to `component` everywhere (here would be `filterComponent`) but I did not follow through with every file yet

Just my opinion, but one of the goals was to be a drop-in replacement, so reducing churn and not needing to rename things (for now) around the `s/_dom.cloneTemplate/_dom.createComponent` is part of the point and helps keep the change slightly smaller. And really they're still documentFragments _containing_ the component (to replicate the old behavior), so I'm personally OK keeping them named `whateverTemplate` until/if we go further down the component road.